src/test/scenarios/datafactory/output/src/datafactory/README.md

```diff
@@ -1,5 +1,95 @@
-Microsoft Azure CLI 'datafactory' Extension
-==========================================
+# Azure CLI datafactory Extension #
+This is a extension for datafactory
 
-This package is for the 'datafactory' extension.
-i.e. 'az datafactory'
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name datafactory
+```
+
+### Included Features ###
+#### datafactory: ####
+datafactory
```

Comment:
seems duplicated. could we only keep on if they are the same?

Reply:
Not dup actually. I try to add some description here. But it seems that swagger file does not contain the description of command group. We just dup the name here, which is the same as it in help,py